site/template.html

```diff
@@ -13,10 +13,10 @@
   <script>try{Typekit.load({ async: true });}catch(e){}</script>
 
   <div class="d-f ai-c p-20 flow-h-20" style="margin-bottom: 120px">
-    <a href="/">
+    <a href="https://mjml.io">
       <img height="40" src="assets/logo-mjml.svg" />
     </a>
-    <a target="_blank" href="https://mjml.io">About</a>
+    <a target="_blank" href="https://github.com/mjmlio/mjml#introduction">About</a>
```

Comment:
Added the link to the MJML website on the MJML logo which seemed more user friendly, so maybe About should link to something else? I linked it to the Readme, what do you think?